app/DataTransferObjects/LicenseManagement/LicenseAddonsData.php

```diff
@@ -50,6 +50,7 @@ public function __construct(
         public bool $knowledgeManagement,
         public bool $realtimeChat,
         public bool $mobileApps,
+        public bool $changeManagement = true,
```

Comment:
Should this be true by default?

Reply:
I don't think you should interact with the settings class in a migration, that kinda breaks our guidelines

Reply:
> Should this be true by default?

In Ticket there is no default behave is defined for this, hence I added true but now changed it to false.